tests/appsec/iast/_test_iast_fixtures.py

```diff
@@ -225,14 +225,14 @@ def validate_request_reported(self, request, source_type=None):
         sources = iast["sources"]
         assert sources, "No source reported"
         if source_type:
```

Comment:
As now those tree parameters are list, can you add this assert ion for all of them : the `in` operator works fine with string, and I can at least one error here : https://github.com/DataDog/system-tests/pull/1805/files#diff-dfdb117911ceb4658243f4b3e1daf9dc6429b8b7952f2e7c04e7dd623112e58bR15 

Code for `source_type` : 

```suggestion
        if source_type:
            assert isinstance(source_type, (tuple, set, list)), "source_type must a set, a tuple or a list"
```

Reply:
They don´t need to be a list, they can be a single string and it will work just fine. I have reverted this assertion after some testing.